app/editor/src/features/admin/sources/schedules/utils/weekDayName.ts

```diff
@@ -0,0 +1,59 @@
+import { WeekDay, WeekDayName } from 'hooks/api-editor';
+
+export const weekDayName = (value: WeekDay | WeekDay[]) => {
```

Comment:
Not a fan of this implementation, but was trying to find a way to reduce the impact of these values on the table row size.